receiver/k8sclusterreceiver/internal/container/containers.go

```diff
@@ -79,6 +79,21 @@ func RecordSpecMetrics(logger *zap.Logger, mb *metadata.MetricsBuilder, c corev1
 			if cs.LastTerminationState.Terminated != nil {
 				rb.SetK8sContainerStatusLastTerminatedReason(cs.LastTerminationState.Terminated.Reason)
 			}
+
+			switch {
+			case cs.State.Running != nil:
```

Comment:
Not for this PR, but at some point we should move those status-related metrics to a different function. This one is called `RecordSpecMetrics` :).